### PR TITLE
refreshing token package

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

File: lusid-jam/refreshing_token.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from collections import UserString
+from datetime import datetime, timedelta, time
+import os
+
+class RefreshingToken(UserString):
+
+    def __init__(self, access_token=os.path.join(os.getenv("FBN_ACCESS_TOKEN_FILE"))):
```
Comment (MikeMcGarry) [inline lusid-jam/refreshing_token.py]: Can we rename this variable to access_token_location please? This makes me think that I am providing the actual token.

Comment (MikeMcGarry) [inline lusid-jam/refreshing_token.py]: Is os.path.join the best thing to call here? We will just be providing the absolute path to the secrets file so I'm not sure if join adds any value as we are not joining it with anything?

**Review by MikeMcGarry — CHANGES_REQUESTED** [review]
Looks good. I have made a few comments.

Can you also please add a few tests to make sure that this works?

There are some in the lusid-sdk-python you can borrow from.

Let's publish it privately and test it to ensure we have the right value to set in the hub for the access token file.

File: lusid-jam/refreshing_token.py (hunk 1 above)
Comment (slemasne) [reply lusid-jam/refreshing_token.py]: Renamed to access_token_location

Comment (slemasne) [reply lusid-jam/refreshing_token.py]: Agreed. The os.path.join method was a typo. I have removed that.

**Comment by slemasne** [comment]
I have added a test to the latest commit.
The test is a very simple test which reads a test token file and returns a string (which represents the LUSID access token)

**Review by MikeMcGarry — APPROVED** [review]

**Pull request merged**